content/en/platform/corda/4.10/enterprise/network/network-reqs.md

```diff
@@ -18,6 +18,9 @@ There is a set of parameters that all Nodes on the network must agree on in orde
 * **minimumPlatformVersion**: 
 The minimum platform version that the nodes must be running. Any node which is below this will not start.
 
+{{< note >}}
+You can set the `minimumPlatformVersion` network parameter to a value that is as high as the Corda Platform version of the nodes we allow to run on a network. For example, we can set `minimumPlatformVersion` to 5 to allow only Corda 4.3+ nodes (Corda Platform version 5+) to run on a network.
+{{< /note >}}
```

Comment:
```suggestion
{{< note >}}
The `minimumPlatformVersion` network parameter can be set to a value high as the Corda Platform nodes' version allowed to run on a network. For example, setting `minimumPlatformVersion` to 5 will allow only Corda 4.3+ nodes (Corda Platform version 5+) to run on a network.
{{< /note >}}
```

Reply:
BTW @Niamh25 - please validate the example.
How is setting minimum of 5.0 will allow corda 4.3+ nodes ?!

Reply:
same comment applies to the other files